dev_dep_versions.yml

```diff
@@ -1,3 +1,7 @@
 __version__: "2.4.0.dev0"
 __cuda_version__: "12.1"
 __tensorrt_version__: "10.0.1"
+__torch_version__: "2.3.0"
```

Comment:
Wrong version

Reply:
Also not sure if this should be in dev deps since main is always on latest nightly

Reply:
removed it. These are redundant changes.